src/test/scala/fr/acinq/bitcoin/samples/FindMyAddresses.scala

```diff
@@ -18,7 +18,7 @@ object FindMyAddresses extends App {
   def address(pub: PublicKey): String = Base58Check.encode(if (testnet) Base58.Prefix.ScriptAddressTestnet else Base58.Prefix.ScriptAddress, Crypto.hash160(Script.write(Script.pay2wpkh(pub))))
 
   // step #1: compute the seed from the mnemonic code
-  val seed = MnemonicCode.toSeed(mnemonics.split(' '), passphrase)
+  val seed = MnemonicCode.toSeed(mnemonics.split(' ').toSeq, passphrase)
```

Comment:
```suggestion
  val seed = MnemonicCode.toSeed(mnemonics, passphrase)
```

Reply:
Fixed in https://github.com/ACINQ/bitcoin-lib/pull/44/commits/7d3622c628c7b1ff51ccf69d7aaae6ce4d664e67